CMakeLists.txt

```diff
@@ -35,35 +35,26 @@ cmake_minimum_required(VERSION 3.5)
 
 set(otf2xx_VERSION 1.0)
 
-set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake;${CMAKE_MODULE_PATH}")
+list(INSERT CMAKE_MODULE_PATH 0 ${CMAKE_CURRENT_LIST_DIR}/cmake)
 find_package(OTF2 8.0.1 EXACT)
 
 if (NOT OTF2_FOUND)
     message(FATAL_ERROR "Please download and install OTF2 2.1.\n"
     "Available from: http://www.vi-hps.org/upload/packages/otf2/otf2-2.1.tar.gz")
 endif()
 
-if(NOT OTF2XX_CHRONO_DURATION_TYPE)
-    set(OTF2XX_CHRONO_DURATION_TYPE "picoseconds")
-endif()
-
-set(OTF2XX_CHRONO_DURATION_TYPE ${OTF2XX_CHRONO_DURATION_TYPE} CACHE STRING "The used underlying type for otf2::chrono::duration.")
-set_property(CACHE OTF2XX_CHRONO_DURATION_TYPE PROPERTY STRINGS "nanoseconds" "picoseconds")
-if(${OTF2XX_CHRONO_DURATION_TYPE} STREQUAL "nanoseconds" OR ${OTF2XX_CHRONO_DURATION_TYPE} STREQUAL "picoseconds")
+set(OTF2XX_CHRONO_DURATION_TYPE "picoseconds" CACHE STRING "The used underlying type for otf2::chrono::duration.")
+set(otf2xx_allowed_duration_types "nanoseconds" "picoseconds")
+set_property(CACHE OTF2XX_CHRONO_DURATION_TYPE PROPERTY STRINGS ${otf2xx_allowed_duration_types})
+if(OTF2XX_CHRONO_DURATION_TYPE IN_LIST otf2xx_allowed_duration_types)
     message(STATUS "OTF2xx uses '${OTF2XX_CHRONO_DURATION_TYPE}' as chrono duration type.")
 else()
-    message(SEND_ERROR "OTF2XX_CHRONO_DURATION_TYPE must be either nanoseconds or picoseconds")
-endif()
-
-if(NOT OTF2XX_WITH_MPI)
-    set(OTF2XX_WITH_MPI OFF)
+    message(SEND_ERROR "OTF2XX_CHRONO_DURATION_TYPE must be one of ${otf2xx_allowed_duration_types}")
 endif()
-option(OTF2XX_WITH_MPI "Whether OTF2xx should be build with MPI support or not. (Requires Boost.MPI)" ${OTF2XX_WITH_MPI})
 
-if(NOT OTF2XX_USE_STATIC_LIBS)
```

Comment:
It might not be the "default" CMake variable, but ..._USE_STATIC_LIBS is used in FindBoost and I think it's also as semi-default way of propagating the wish for static linkage to dependencies. For the sake of compatibility, I'd opt for keeping it the way it is.